groups/rds/data.tf

```diff
@@ -29,5 +29,5 @@ data "aws_subnet_ids" "placement" {
 
 data "aws_route53_zone" "selected" {
   name         = local.dns_zone_name
-  private_zone = false
+  private_zone = true
```

Comment:
This should be variable as whether the route53 zone is private depends on the account we're deploying in to. E.g. in development, it's public. In shared-services it's private.

Reply:
Implemented via commit: 252d107678658864551f07a8c8958fea666199bc